packages/react-composites/src/mocks/MockChatClient.ts

```diff
@@ -2,21 +2,24 @@
 // Licensed under the MIT License.
 
 import { ChatClient, ChatThreadClient } from '@azure/communication-chat';
-
+/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+import { ChatThreadItem, CreateChatThreadResult } from '@azure/communication-chat';
 import {
   StatefulChatClient,
   StatefulChatClientArgs,
   _createStatefulChatClientWithDeps
 } from '@internal/chat-stateful-client';
+/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+import { ChatClientState, ChatErrors } from '@internal/chat-stateful-client';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
 
 type PublicInterface<T> = { [K in keyof T]: T[K] };
 /**
  * @returns
  */
-export const createStatefulChatClientMock = (threadClient: PublicInterface<ChatThreadClient>): StatefulChatClient => {
+export function createStatefulChatClientMock(threadClient: PublicInterface<ChatThreadClient>): StatefulChatClient {
```

Comment:
Any reason for not using the arrow function syntax here?

Reply:
Reverted